kloppy/infra/serializers/tracking/epts/metadata.py

```diff
@@ -242,5 +242,6 @@ def load_metadata(metadata_file: Readable) -> EPTSMetadata:
         sensors=sensors,
         score=score,
         orientation=None,
+        provider=None,
```

Comment:
@bdagnino what should we do here? EPTS isn't a provider

Reply:
@koenvo I think it would be good to have the provider as an input to load_metadata with a default to None, like:

```python
def load_metadata(metadata_file: Readable, procider: Provider = None) -> EPTSMetadata:
```

At the moment it assumes it's a Metrica data file (because of ProviderParameters it uses), but eventually it will need to handle different ProviderParameters, depending on the provider, so it'd be good to already accept this. 

What do you think?

Reply:
I agree with that, I have added the default = None to epts/metadata.py 

I think it is all ready for final review and hopefully merge

Reply:
Until we have a better way to determine the provider of a EPTS lets go with the None value.  I'll merge the request. Thanks @FCrSTATS